src/main/java/reactor/netty/http/server/HttpServerResponse.java

```diff
@@ -156,75 +156,12 @@
-	 * is confirmed, then the provided callback is invoked, if the upgrade is not
-	 * successful the returned {@link Mono} fails.
-	 *
-	 * @param proxyPing whether to proxy websocket ping frames or respond to them
-	 * @param websocketHandler the I/O handler for websocket transport
-	 * @return a {@link Mono} completing when upgrade is confirmed, otherwise fails
-	 */
-	default Mono<Void> sendWebsocket(boolean proxyPing,
-			BiFunction<? super WebsocketInbound, ? super WebsocketOutbound, ? extends Publisher<Void>> websocketHandler) {
-		return sendWebsocket(null, 65536, proxyPing, websocketHandler);
-	}
-
-	/**
-	 * Upgrades the connection to websocket with optional subprotocol(s). A {@link Mono}
-	 * completing when the upgrade is confirmed, then the provided callback is invoked,
-	 * if the upgrade is not successful the returned {@link Mono} fails.
-	 *
-	 * @param protocols optional sub-protocol
-	 * @param websocketHandler the I/O handler for websocket transport
-	 *
-	 * @return a {@link Mono} completing when upgrade is confirmed, otherwise fails
-	 */
-	default Mono<Void> sendWebsocket(@Nullable String protocols,
-			BiFunction<? super WebsocketInbound, ? super WebsocketOutbound, ? extends Publisher<Void>> websocketHandler) {
-		return sendWebsocket(protocols, 65536, websocketHandler);
-	}
-
-	 /**
-	 * Upgrades the connection to websocket with optional subprotocol(s). A {@link Mono}
-	  * completing when the upgrade is confirmed, then the provided callback is invoked,
-	  * if the upgrade is not successful the returned {@link Mono} fails.
-	 *
-	 * @param protocols optional sub-protocol
-	 * @param maxFramePayloadLength maximum allowable frame payload length
-	 * @param websocketHandler the I/O handler for websocket transport
-	 *
-	 * @return a {@link Mono} completing when upgrade is confirmed, otherwise fails
-	 */
-	default Mono<Void> sendWebsocket(@Nullable String protocols,
-			int maxFramePayloadLength,
-			BiFunction<? super WebsocketInbound, ? super WebsocketOutbound, ? extends Publisher<Void>> websocketHandler) {
-		return sendWebsocket(protocols, maxFramePayloadLength, false, websocketHandler);
-	}
-
-	/**
-	 * Upgrades the connection to websocket with optional subprotocol(s). A {@link Mono}
-	 * completing when the upgrade is confirmed, then the provided callback is invoked,
-	 * if the upgrade is not successful the returned {@link Mono} fails.
-	 *
-	 * @param protocols optional sub-protocol
-	 * @param maxFramePayloadLength maximum allowable frame payload length
-	 * @param proxyPing whether to proxy websocket ping frames or respond to them
-	 * @param websocketHandler the I/O handler for websocket transport
-	 *
-	 * @return a {@link Mono} completing when upgrade is confirmed, otherwise fails
-	 */
-	Mono<Void> sendWebsocket(@Nullable String protocols,
-			int maxFramePayloadLength,
-			boolean proxyPing,
-			BiFunction<? super WebsocketInbound, ? super WebsocketOutbound, ? extends Publisher<Void>> websocketHandler);
-
```

Comment:
Please keep those methods as deprecated.
For version 0.9.x they will stay deprecated and for version 1.0.x we will remove them.